aten/src/THC/THCApply.cuh

```diff
@@ -221,7 +221,7 @@ bool THC_pointwiseApply1(THCState* state,
       THCTensor_maybeOverlappingIndices(state, a)) {
     // Must perform in contiguous space
     oldA = a;
-    a = TensorUtils<TensorTypeA>::newContiguous(state, a);
+    a = (TensorTypeA*)THCTensor_newContiguous<ScalarTypeA>(state, a);
```

Comment:
A C cast!! You're so naughty ;)

Reply:
That should be easy to get rid of -- TensorTypeA is just THCTensor now.